cmd/create/cluster/cmd.go

```diff
@@ -517,6 +501,16 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	if args.dryRun {
+		reporter.Infof(
+			"Creating cluster '%s' should succeed. Run without the '--dry-run' flag to create the cluster.",
+			clusterName)
+		os.Exit(0)
+	}
```

Comment:
This block needs to be moved back down, it should only display if `CreateCluster` succeeded.